web/src/test/java/com/replica/StreetEdgeExporterTest.java

```diff
@@ -38,36 +40,72 @@ public void testExportEndToEnd() throws IOException {
         int emptyNodeIdCount = 0;
         int emptyWayIdCount = 0;
         Set<String> observedStableEdgeIds = Sets.newHashSet();
+        Set<String> observedHumanReadableStableEdgeIds = Sets.newHashSet();
 
         // Remove header row
         records.remove(0);
 
-        // Remove small number of non-unique rows in output (expected due to OSM node ID parsing method).
```

Comment:
Note that we can remove this operation, because we no longer have any dupes in our raw output, unlike before

Reply:
Exactly, we're getting rid of it there ([here's](https://github.com/replicahq/model/pull/6562/files#diff-f851fba0ffb454973f43f97bc0ceb65c056c0d4915d7f9128ea907e965889221L59) the change in the in-flight core update branch). All the full-table QA I did was with the model-repo uniquing removed, so we know that we get full nationwide unique IDs even without it (this java uniquing code is only for the unit test, to replicate that behavior)

Reply:
also, for your second question - we never really figured it out 🤔 that was one remaining issue from our approach for parsing OSM node IDs, which was what introduced all this hacky osm reading code in the first place. Because the number of dupes was so small we just accepted it + filtered them out